### ISSUE TITLE
Server files missing 0.2.7 

### ISSUE THREAD

**Issue by Zekirax** [AllTheMods/Gravitas2#214]
I think  the Server files was forgotten is the Gravitas2-0.2.7 release. Same for the hotfix. Here , there is no Additional Files. [Additional Files](https://www.curseforge.com/minecraft/modpacks/all-the-mods-gravitas2/files/5201172/additional-files)

**Comment by Geldra** [comment]
I was able to work around this by using [The last available server download](https://www.curseforge.com/minecraft/modpacks/all-the-mods-gravitas2/files/5165113) and then overwriting with two main operations:
1. I removed all of the mods on the server and placed all of the client mods (removing `oculus-*`)
2. I unzipped the 0.2.7 client folder and moved everything in `overrides` into the server. (overwriting the existing files from installing the previous version)
